packages/website/ts/index.tsx

```diff
@@ -86,7 +86,10 @@ render(
                             <Route path={WebsitePaths.Wiki} component={Wiki as any} />
                             <Route path={`${WebsitePaths.ZeroExJs}/:version?`} component={LazyZeroExJSDocumentation} />
                             <Route path={`${WebsitePaths.Connect}/:version?`} component={LazyConnectDocumentation} />
-                            <Route path={`${WebsitePaths.Deployer}/:version?`} component={LazyDeployerDocumentation} />
+                            <Route
+                                path={`${WebsitePaths.SolCompiler}/:version?`}
+                                component={LazySolCompilerDocumentation}
+                            />
```

Comment:
We should keep the `/docs/deployer` url around and re-direct it to `docs/sol-compiler`. See `Legacy endpoints` section below.